.github/workflows/testCoverage.yml

```diff
@@ -253,58 +215,46 @@ jobs:
                 echo "$statementCovJson" > ubuntuCoverage.json
             fi
           done
+          
         shell: bash
 
-      - name: Change branch to test-coverage-artifacts and move new artifacts to folder
+      - name: Move coverage JSONs to root
         run: |
+          mv -f ./repo/.github ./
           cd repo
-          if [ `git branch | grep test-coverage-artifacts` ]
-          then
-          git checkout test-coverage-artifacts
-          git pull origin test-coverage-artifacts
-          else
-          git checkout -b test-coverage-artifacts
-          fi
           rm -rf ./*
           cd ..
+          mv -f ./.github ./repo/
           mv -f ./coverage/* ./repo/
+          cd repo
+          ls
       - uses: stefanzweifel/git-auto-commit-action@v5
         with:
           repository: repo
           branch: test-coverage-artifacts
-          push_options: "--force"
-        
-          
-  deploy:
-    needs: push_artifacts_to_repo
-    runs-on: "ubuntu-latest"
+  
+
+  build_and_deploy_job:
+    needs: [push_artifacts_to_repo]
```

Comment:
are you moving logic from azure CI workflow to this workflow?

Reply:
Yes

Reply:
don't do that. Just call that workflow for now